tests/cypress/integration/devportal/000-general/04-change-password-of-a-user.spec.js

```diff
@@ -52,7 +52,7 @@ describe("Change the password from devportal", () => {
     cy.get("input#repeated-new-password").click();
     cy.get("input#repeated-new-password").type(newPassword);
 
-    cy.get("button > span").contains("Save").click();
+    cy.get("button").contains("Save").click();
```

Comment:
Better to add the data test id in the ui for elements selection
```suggestion
    cy.get('[data-testid="change-password-save-button"]').contains("Save").click();
```

Reply:
Resolved by the commit 663b7de